surveys/2025/annual-survey/translations/de.md

```diff
@@ -469,14 +469,14 @@ Rows:
 
 - Informationsmaterial über Rust erstellen (z.B. einen Blog schreiben, streamen, YouTube Videos produzieren, auf einer Konferenz präsentieren, …)
 - Informationsmaterial über Rust konsumieren (z.B. eine Artikel lesen, Streams oder Videos schauen, …)
-- Offizielle Kommunikationskanälen lesen (z.B. This Week in Rust, den offizielle Rust Blog, den Twitteraccount von Rust, …)
+- Offizielle Kommunikationskanälen lesen (z.B. This Week in Rust, den offiziellen Rust Blog, den Bluesky Account von Rust, …)
 - An Unterhaltungen auf Social Media über Rust teilnehmen (z.B. auf Hacker News, r/rust, Twitter, LinkedIn, …)
 - Ein Rust Meetup oder eine Konferenz (virtuell oder vor Ort) besuchen
-- Einen RFC schreiben, kommentieren, zu der Diskussion betragen, oder Änderungen an dem RFC vorschlagen
-- Das Rust-Projekt in einem offiziellen Chat besprechen (entweder Zulip oder Discord)
+- Einen RFC schreiben, kommentieren, zu der Diskussion beitragen, oder Änderungen an einem RFC vorschlagen
+- Das Rust-Projekt in einem offiziellen Chat oder Forum besprechen (z.B. internals.rust-lang.org, Rust Zulip, …)
 - Ein Issue in einem beliebigen Repository in der rust-lang GitHub-Organisation eröffnen
-- Code-Änderungen (einschließlich Tests) zu einem beliebigen Projekt in der rust-lang GitHub-Organisation beitragen
-- Nicht-Code-Änderungen (Dokumentation, Kommentare usw.) zu einem Projekt in der rust-lang GitHub-Organisation beitragen
+- Code-Änderungen (einschließlich Tests) zu einem beliebigen Open-Source Rust Projekt beitragen
+- Nicht-Code-Änderungen (Dokumentation, Kommentare usw.) zu einem beliebigen Open-Source Rust Projekt beitragen
```

Comment:
The English questions refer to "any open-source Rust project" here as well:
https://github.com/rust-lang/surveys/blob/d3532ca43a0d8803a429328581416ed003dc7456/surveys/2025/annual-survey/questions.md?plain=1#L524-L525

Not sure though if they imply the rust-lang Github org?